Documentation/WritingDocForExtension/Migrate.rst

```diff
@@ -32,32 +32,222 @@ a migration tool. This tool can be used to automatically create a
 :file:`Documentation/guides.xml` from the information contained in your
 :file:`Documentation/Settings.cfg`.
 
-`Docker <https://docs.docker.com/install/>`__ needs to be installed on your
+`Docker <https://docs.docker.com/install/>`__ (or a drop-in replacement like
+`Podman <https://podman.io>`__) needs to be installed on your
 operating system for the tool to work:
 
-..  code-block:: shell
+..  tabs::
+
+    ..  group-tab:: Docker
+
+        ..  code-block:: bash
+
+            docker run --rm --pull always \
+              -v $(pwd):/project \
+              -it ghcr.io/typo3-documentation/render-guides:latest \
+              migrate Documentation
+
+    .. group-tab:: Podman
 
-    docker run --rm --pull always \
-        -v $(pwd):/project \
-        -it ghcr.io/typo3-documentation/render-guides:latest \
-        migrate Documentation
+        ..  code-block:: bash
+
+            podman run --rm --pull always \
+              -v $(pwd):/project \
+              -it ghcr.io/typo3-documentation/render-guides:latest \
+              migrate Documentation
+
+
+The last parameter (:bash:`Documentation`) is the name of the directory, where your
+:file:`Settings.cfg` is currently placed in.
 
 Try out the rendering locally
 =============================
 
 Use our Docker container to render your documentation locally. Read more about
 :ref:`local rendering <rendering-docs>`.
 
-Improve your documentation to render without warning
-====================================================
+..  _migrate-detailed-steps:
+
+Further steps to adapt to the new rendering
+===========================================
 
-If you believe you found a bug in the new PHP-based rendering, please open
+You should perform the following tasks to conclude the migration to the
+new rendering tool:
+
+1. Improve your documentation to render without warning
+-------------------------------------------------------
+
+Rendering your documentation should not yield any warnings or errors.
+
+If you get error messages, often they refer to wrong indentation, missing
+interlinks, orphaned files or outdated ReST-identifiers.
+
+If you are unable to address a warning/error with changes in your documentation
+and if you believe you found a bug in the new PHP-based rendering, please open
 an `Issue on GitHub <https://github.com/TYPO3-Documentation/render-guides/issues>`__.
 
-Recommended: Activate automatic testing
-=======================================
+2. Remove outdated files
+-----------------------------------
+
+After you have created the :file:`guides.xml` file, you can remove the old
+:file:`Settings.cfg` file.
+
+You can also delete the files :file:`genindex.rst` which was previously used
+to generate an index.
```

Comment:
The reference in the Documentation/Index.rst file should then also removed.

Reply:
That's mentioned later on.

Reply:
It's not twice.

1: Remove genindex.rst file
2: Remove genindex reference

That's two different tasks